integration/integrate-with-slack.adoc

```diff
@@ -0,0 +1,24 @@
+[id="integrate-with-slack"]
+= Integrating with Slack
+include::modules/common-attributes.adoc[]
+:context: integrate-with-slack
+
+toc::[]
+
+[role="_abstract"]
+If you are using https://slack.com/[Slack], you can forward alerts from {product-title} to Slack.
```

Comment:
I don't think the Slack link is necessary here and probably best to avoid links to external sites unless it is for additional problem solving. 🤷